acra-dialog/src/main/java/org/acra/dialog/CrashReportDialog.kt

```diff
@@ -232,4 +232,4 @@ open class CrashReportDialog : Activity(), DialogInterface.OnClickListener {
         private const val STATE_EMAIL = "email"
         private const val STATE_COMMENT = "comment"
     }
-}
\ No newline at end of file
+}
```

Comment:
sorry about this, it was just a GitHub Web UI PR and they like newlines at end of file I guess
If this is bothersome let me know of course